api/package.json

```diff
@@ -16,8 +16,12 @@
         "seed": "ts-node -r tsconfig-paths/register prisma/seed.ts"
     },
     "scripts": {
+        "build": "swc src -d dist --copy-files -s false",
         "prepare": "cd .. && husky install",
-        "start:local": "nodemon --config nodemon.json",
+        "dev": "concurrently npm:dev:start npm:dev:watch npm:dev:type",
+        "dev:start": "nodemon --config nodemon.json",
```

Comment:
You don't need nodemon now. SWC is watching for changes so will recompile the changes. The command you're having nodemon run includes the `reloadHandler` argument for serverless, so it will pick up the latest changes. Can save yourself a config file and a dependency

Reply:
That makes sense, just removed it and local dev still working fine.